packages/components/src/ui/ariakit-tooltip/test/index.tsx

```diff
@@ -0,0 +1,301 @@
+/**
+ * External dependencies
+ */
+import { render, screen, waitFor } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+
+/**
+ * WordPress dependencies
+ */
+import { shortcutAriaLabel } from '@wordpress/keycodes';
+
+/**
+ * Internal dependencies
+ */
+import Button from '../../../button';
+import { ToolTip } from '..';
+import { TOOLTIP_DELAY } from '../../../tooltip/';
+
+const props = {
+	children: <Button>Button</Button>,
+	text: 'tooltip text',
+	delay: TOOLTIP_DELAY,
+};
+
+describe( 'ToolTip', () => {
+	it( 'should not render the tooltip if multiple children are passed', async () => {
+		const user = userEvent.setup();
+		// mock expected console error
+		const err = jest
+			.spyOn( console, 'error' )
+			.mockImplementation( () => {} );
+
+		expect( () => {
+			render(
+				// expected TS error since Tooltip cannot have more than one child element
+				// @ts-expect-error
+				<ToolTip { ...props }>
+					<Button>This is a button</Button>
+					<Button>This is another button</Button>
+				</ToolTip>
+			);
+		} ).toThrow();
+
+		expect( console ).toHaveErrored();
+		expect( err ).toHaveBeenCalled();
+
+		await user.tab();
+
+		expect(
+			screen.queryByRole( 'tooltip', { name: /tooltip text/i } )
+		).not.toBeInTheDocument();
+
+		err.mockReset();
+	} );
+
+	it( 'should not render the tooltip if there is no focus', () => {
+		render( <ToolTip { ...props } /> );
+
+		expect(
+			screen.getByRole( 'button', { name: /Button/i } )
+		).toBeVisible();
+
+		expect(
+			screen.queryByRole( 'tooltip', { name: /tooltip text/i } )
+		).not.toBeInTheDocument();
+	} );
+
+	it( 'should render the tooltip when focusing on the tooltip anchor via tab', async () => {
+		const user = userEvent.setup();
+
+		render( <ToolTip { ...props } /> );
+
+		const button = screen.getByRole( 'button', { name: /Button/i } );
+
+		await user.tab();
+
+		expect( button ).toHaveFocus();
+
+		await waitFor( () =>
+			expect(
+				screen.getByRole( 'tooltip', { name: /tooltip text/i } )
+			).toBeVisible()
+		);
+
+		await user.tab();
+		await user.unhover( button );
```

Comment:
After tabbing away from the child element, the tooltip should disappear. The only way I could get it to pass was to 'unhover' the button after tabbing, which is quite strange... the behaviour appears to work differently than what I see from manually testing this process in the browser.

Reply:
As I mentioned in my earlier comment, when testing in storybook the behavior seems to match what the test sees - tabbing doesn't hide the tooltip, at least in my testing in the latest Chrome.